extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -236,6 +244,24 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
     super.onCreativeRender(isVerifiedAmpCreative);
     injectActiveViewAmpAnalyticsElement(
       this, this.extensions_, this.ampAnalyticsConfig);
+    this.centerCreative_();
```

Comment:
Not sure that `onCreativeRender` is the best place to put this.  Feels like it should go closer to the iframe creations -- there's already some styling stuff there.